dataGenerate.js

```diff
@@ -0,0 +1,38 @@
+import short from "short-uuid";
+import fs from "fs";
+
+class book {
```

Comment:
class name should start with capital letter.

Reply:
done